src/components/Card/Card.tsx

```diff
@@ -11,18 +12,41 @@ function Card({
   flip: boolean;
   onClick: () => void;
 }) {
+  const ref = React.useRef<HTMLDivElement>(document.createElement("div"));
+
+  React.useEffect(() => {
+    let entryPoint = [0, 0];
+    const track = (moveEvent: MouseEvent) => {
+      moveEvent.stopPropagation();
+      ref.current.style.transform = `translate(${
+        (moveEvent.offsetX - entryPoint[0]) * 0.02
+      }px, ${(moveEvent.offsetY - entryPoint[1]) * 0.02}px)`;
+    };
+
+    ref.current.addEventListener("mouseenter", (event) => {
+      entryPoint = [event.offsetX, event.offsetY];
+      ref.current.addEventListener("mousemove", track);
+    });
+
+    ref.current.addEventListener("mouseleave", () => {
+      ref.current.removeEventListener("mousemove", track);
+      ref.current.removeAttribute("style");
+    });
+  }, [ref]);
+
   return (
-    <div
-      onClick={onClick}
-      className={`${styles.card} ${flip ? styles.flipped : ""} ${
-        off ? styles.off : ""
-      }`}
-    >
-      <div className={styles.front}>
-        <img className={styles.image} src={imageUrl} />
-      </div>
-      <div className={styles.back}>
-        <img className={styles.image} src={back} />
+    <div className={styles.cardholder} ref={ref} onClick={onClick}>
```

Comment:
`ref={cardHolderRef}` would be better - you'll now what it refers to, and you don't need to change code if you create another ref here.